gramofonsite/migrations/0015_rename_morska_title_local_dance_room_title_and_more.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 4.2.2 on 2023-07-18 22:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('gramofonsite', '0014_rename_image_local_image_base_and_more'),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
tu też jest rename, możemy to usunąć. Najlepiej było by chyba usunąć wszystkie migracje i wygenerować nowe od początku